### ISSUE TITLE
Add feature to check cache stored

### ISSUE THREAD

**Issue by Try4W** [vincentbrison/dualcache#27]
Something like

`dualCache.containsData();`

**Comment by Try4W** [comment]
Can I do it this way for now?
```
    @Override
    public boolean isCached() {
        return dualCache.get(лун) != null;
    }
```

**Comment by vincentbrison** [comment]
Totally

**Comment by mack369** [comment]
@Try4W it does however introduce some performance overhead. It deserializes the object from the disc cache which is a waste of computation power

**Comment by vincentbrison** [comment]
Thats true, I will add this feature in the next release ;)

**Comment by vincentbrison** [comment]
Released in 3.1.0
